core/src/wallet/ripple/RippleLikeAccount.cpp

```diff
@@ -123,9 +123,18 @@ namespace ledger {
             operation.trust = std::make_shared<TrustIndicator>();
             operation.date = transaction.receivedAt;
 
+            auto setAmount = [&](Operation& operation) {
+                if (transaction.status == "tesSUCCESS") {
```

Comment:
Maybe use an enum for that : https://xrpl.org/transaction-results.html ?

Reply:
Yeah so there are several ways to go here. They have the concept of a _status code_, but I don’t really understood how I’m supposed to parse that. :/

I don’t really know which way is the best. An enum can be cool, I agree, even though I don’t really know how it would react with `soci`. An integer-based enum, maybe?